docs/cli.rst

```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+You need to edit the specific activate script for different shell, for example:
```

Comment:
There are different activate script in there (e.g. activate, activate.csh, activate.fish...) for different shell.